lib/routemaster/redis_broker.rb

```diff
@@ -25,18 +27,42 @@ def cleanup
       _cleanup
     end
 
+    # Allow to inject pre-built Redis clients
+    #
+    # Before storing a new connection, ensure that any previously
+    # set client is properly closed.
+    #
+    def inject(clients={})
```

Comment:
I can do three things:

1. Refactor how `Config` and `RedisBroker` interact, so that I can more sanely move the interfact to `Config`;
2. I can add public API to the `Config` module that will just delegate to what I've already done;
3. I can leave it as it is.